app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    action, source_file, destination_file = command.split()
+
+    if action != "cp" or len(command.split()) != 3:
```

Comment:
would make sense to check len of command split before you actually split and unpack - otherwise you might get error when unpacking